shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -459,4 +477,29 @@ - (void)unregisterTexture:(int64_t)textureID {
   [_textures removeObjectForKey:@(textureID)];
 }
 
+#pragma mark - Task runner integration
+
+- (void)postMainThreadTask:(FlutterTask)task targetTimeNanos:(uint64_t)target_time {
```

Comment:
macOS framework style is to pre-declare at the top of the file, with a declaration comment, all private methods.